src/main/java/mate/academy/service/impl/AuthenticationServiceImpl.java

```diff
@@ -37,9 +37,9 @@ public User register(String email, String password) throws RegistrationException
         return userService.add(user);
     }
 
-    private boolean passwordMatches(Optional<User> optionalUser, String password) {
+    private boolean passwordMatches(Optional<User> userFromDb, String password) {
```

Comment:
better pass user instead of optional to the parameters
```suggestion
    private boolean passwordMatches(User userFromDb, String password) {
```